README.md

```diff
@@ -13,34 +13,31 @@ In this lab, we'll be dealing with two sets of classes. The `Artist` and `Song`
 
 ### Building the has many/belongs to relationship
 
-How can an artist have many songs and a song belong to an artist? An individual song will need to have an `artist=()` and `artist` method and an individual artist will need to be initialized with an empty collection of songs. 
+How can an artist have many songs and a song belong to an artist? 
 
-How can an author have many posts and a post belong to an author? An individual post will need to have an `author=()` and `author` method and an individual author will need to be initialized with an empty collection of posts.
+An individual song will need to be able to be initialized with an `artist` object -- which can be passed in as an argument to the Song.new method -- and should have an `#artist` getter method to reveal the artist. The Song class itself should have a class variable `@@all` equal to an array which will contain each created Song, the songs should be added to that array when they are created. It should be able to reveal that array of songs with an `.all` class method.
 
-#### The `#add_song_by_name` and `#add_post_by_title` Methods
+An individual artist will need to have a `#songs` method that will run a `select` iteration on the return value of the Song classes `all` method (The array of songs contained in its `@@all` array), and select the songs whose artist is equal to `self`.
+
+How can an author have many posts and a post belong to an author? Well by using the same exact logic we used for our Song and Artist classes! The Post class will keep track of all created posts in an `@@all` class variable, exposed by it's `all` getter method. And each instance of post will hold on to it's author. An author will be able to find it's posts by calling the Post classes `all` method, and running a `select` iteration on that array to capture each post with an author equal to self.
 
-To get the tests on these methods passing, you'll need to define methods that take in an argument of a name (or title), use that argument to create a new song (or post) and *then* associate the objects. 
+#### The `#add_song_by_name` and `#add_post_by_title` Methods
 
-For `add_post_by_title`, you'll want to make sure the body of the method associates the newly created post with an author and adds the post to the author's collection.
+To get the tests on these methods passing, you'll need to define methods that take in an argument of a name (or title), and use that argument to create a new song (or post) passing in `self` as the artist (or author) which will associate the objects to each other. 
 
 #### The `#artist_name` and `#author_name` Methods
 
 Build the methods `some_song.artist_name` and `some_post.author_name` that return the names of the artist and author of the given song and post respectively.
```

Comment:
I know this isn't part of the changes, but we should use Ruby's documentation syntax here: `Song#artist_name` and `Post#author_name`.